tests/test_date_parsers.py

```
@@ -19,353 +19,335 @@
+        (366, ', not tomorrow')
 
-        # Check date validator
-        self.assertRaises(JVF, days_in_strings, 'Today=(2001-11-31)', first=True)
-        self.assertRaises(JVF, days_in_strings, 'Today=(2001-11-31)', first=False)
-        self.assertRaises(JVF, days_in_strings, 'Today 2001-02-29T12:34:56', first=True)
-        self.assertRaises(JVF, days_in_strings, 'Today 2001-02-29T12:34:56', first=False)
-        self.assertEqual(day_in_string('Today 2001-01-01, not tomorrow', remainder=True),
-                                       (366, ', not tomorrow'))
+    # Real-world test
+    test_path = 'test_files/cpck15Dec2017.tpc'
+    with open(test_path, 'r') as f:
```

Ruff (0.15.6)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)